pkg/detector/ospkg/redhat/redhat.go

```diff
@@ -4,16 +4,14 @@ import (
 	"strings"
 	"time"
 
-	"golang.org/x/xerrors"
+	"github.com/aquasecurity/trivy-db/pkg/vulnsrc/redhat"
```

Comment:
Also, I think we can merge it into the group in L12-

Reply:
Done